lib/log_displayer.js

```diff
@@ -44,7 +44,8 @@ function readLogs (logplexURL) {
   return new Promise(function (resolve, reject) {
     let u = url.parse(logplexURL, true)
     let isTail = u.query.tail && u.query.tail === 'true'
-    let es = new EventSource(logplexURL)
+    let userAgent = process.env.HEROKU_DEBUG_USER_AGENT || 'heroku-run';
```

Comment:
Good call, but can we use the "official" user-agent string that gets sent to API? On my machine this is
>     user-agent=heroku-cli/6.15.26-5726b6f (linux-x64) node-v9.5.0
I'm not sure where it's coming from--do you know @jdxcode?

Reply:
Ah sure. Yeah would be good to know how to fetch that.

Reply:
depending on when a plugin was written the user agent is set in very different places. This is a very old plugin and it's being set here: https://github.com/heroku/heroku-cli-util/blob/764b02bb53ff88cbe391c2d7f8b89d60380acb36/lib/command.js#L87